normandy/recipes/checks.py

```diff
@@ -102,20 +105,32 @@ def signatures_use_good_certificates(app_configs, **kwargs):
         msg = f"Could not retrieve signatures: {e}"
         errors.append(Info(msg, id=INFO_COULD_NOT_RETRIEVE_SIGNATURES))
     else:
+
+        def get_matching_object_names(url):
+            matching_recipes = Recipe.objects.filter(signature__x5u=url)
+            matching_actions = Action.objects.filter(signature__x5u=url)
+            matching_objects = list(matching_recipes) + list(matching_actions)
+            object_names = ", ".join(str(o) for o in matching_objects)
+            return object_names
+
         for url in urls:
             try:
                 signing.verify_x5u(url, expire_early)
             except signing.BadCertificate as exc:
-                matching_recipes = Recipe.objects.filter(signature__x5u=url)
-                matching_actions = Action.objects.filter(signature__x5u=url)
-                bad_objects = list(matching_recipes) + list(matching_actions)
-
-                object_names = ", ".join(str(o) for o in bad_objects)
+                bad_object_names = get_matching_object_names(url)
+                msg = (
+                    f"{len(bad_object_names)} objects are signed with a bad cert: "
+                    f"{bad_object_names}. {exc.detail}. Certificate url is {url}. "
+                )
+                errors.append(Error(msg, id=ERROR_BAD_SIGNING_CERTIFICATE))
+            except requests.ConnectionError as exc:
```

Comment:
Is it only `ConnectionError`s we've seen in Sentry? We could consider using `RequestException`.

Reply:
That would include things like `Timeout`.

Reply:
Actually, there have been at least [2 `HTTPError`s](https://sentry.prod.mozaws.net/operations/normandy-stage/issues/4951178/?query=is:unresolved) that would be missed by `ConnectionError`. 

Seems the simplest and safest is to just use `except requests.RequestException as exc:`